lib/Backend/CodeGenNumberAllocator.cpp

```diff
@@ -333,6 +333,7 @@ Js::JavascriptNumber* XProcNumberPageSegmentImpl::AllocateNumber(Func* func, dou
             if (!WriteProcessMemory(hProcess, (void*)number, pLocalNumber, sizeCat, &bytesWritten)
                 || bytesWritten != sizeCat)
             {
+                Js::Throw::CheckAndThrowJITOperationFailed();
                 Output::Print(_u("FATAL ERROR: WriteProcessMemory failed, GLE: %d\n"), GetLastError());
```

Comment:
done

---
In reply to: [81823110](https://github.com/Microsoft/ChakraCore/pull/1689#discussion_r81823110) [](ancestors = 81823110)